snykTask/src/index.ts

```diff
@@ -402,7 +399,11 @@ async function run() {
       snykTestResult.code === CLI_EXIT_CODE_ISSUES_FOUND &&
       taskArgs.failOnIssues
     ) {
-      throw new SnykError(snykTestResult.message);
+      let failureThreshold:string = taskArgs.failOnThreshold!
```

Comment:
You won't need non-null assertion operator here once you remove `undefined` declaration in the property definition (see my other comment).